packages/rx_bloc_cli/mason_templates/bricks/lib_auth_matrix/__brick__/lib/feature_auth_matrix/services/auth_matrix_edit_address_service.dart

```diff
@@ -0,0 +1,91 @@
+{{> licence.dart }}
+
+import 'package:widget_toolkit/edit_address.dart';
+
+import '../../assets.dart';
+import '../../base/models/errors/error_model.dart';
+
+class AuthMatrixEditAddressService extends EditAddressService<CountryModel> {
+  AuthMatrixEditAddressService();
+
+  Future<List<CountryModel>> get searchList => Future.delayed(
+        const Duration(seconds: 1),
+        () => _countriesList
+            .map((country) =>
+                CountryModel(countryCode: 'US', countryName: country))
+            .toList(),
+      );
+
+  final _countriesList = [
+    'Angola',
+    'Bulgaria',
+    'Cuba',
+    'Egypt',
+    'Italy',
+    'Angola',
+    'Bulgaria',
+    'Cuba',
+    'Egypt',
+    'Italy',
+    'Angola',
+    'Bulgaria',
+    'Cuba',
+    'Egypt',
+    'Italy'
+  ];
+
+  @override
+  Future<AddressModel> saveAddress(AddressModel addressModel) async {
+    await Future.delayed(const Duration(seconds: 1));
+    return addressModel;
+  }
+
+  @override
+  Future<List<CountryModel>> getCountries() async => await searchList;
```

Comment:
Do we need this `async => await` here? Seems redundant

Reply:
No, it's not needed.